cedar-policy-core/src/evaluator/err.rs

```diff
@@ -83,24 +83,26 @@ pub enum EvaluationError {
     InvalidRestrictedExpression(#[from] RestrictedExpressionError),
 
     /// Thrown when a policy is evaluated with an un-filled slot
-    #[error("Template slot {0} was not instantiated")]
+    #[error("template slot `{0}` was not filled")]
     TemplateInstantiationError(SlotId),
 
     /// Evaluation error thrown by an extension function
-    #[error("error from {extension_name} extension: {msg}")]
+    #[error("error while evaluating {extension_name} extension function: {msg}")]
```

Comment:
`ExtensionError` to `ExtensionAppError`? And `{extension_name}` to "\`{extension_name}\`"?